k8s.gcr.io/images/k8s-staging-build-image/images.yaml

```diff
@@ -100,5 +100,5 @@
     - v12.0.1
 - name: go-runner
   dmap:
-    sha256:536ab131b0d0e3b13eb83c985cc0ac9ba7e69e7dac9521e6cacd6a8b6019e0a6:
+    sha256:8ee20934e6c005a9ce8d6d8b7ed23698c3bb80e0b30a3d49e5aeca928cc69bf3:
```

Comment:
We have to revert this. We shouldn't/can't modify tags in prod.

Reply:
resolved on slack conversation - https://kubernetes.slack.com/archives/C2C40FMNF/p1588871803210600?thread_ts=1588868307.203500&cid=C2C40FMNF